odk1-src/form-question-types.rst

```diff
@@ -1275,16 +1287,16 @@ type
 appearance
   :tc:`placement-map`
 
-The default :ref:`geopoint-widget` does not allow the user to select a location.
+The default :ref:`geopoint-widget` does not allow the user to select a location other than the device's current geolocation.
```

Comment:
select a location other than -> place the point anywhere other than

Reply:
Eagle-eyed @zestyping!